lambdas/data-migration2/tests/test-executions.js

```diff
@@ -501,16 +513,27 @@ test.serial('migrateExecutions processes all non-failing records', async (t) =>
 
 test.serial('migrateExecutions logs summary of migration for a specified loggingInterval', async (t) => {
   const logSpy = sinon.spy(Logger.prototype, 'info');
-  process.env.loggingInterval = 1;
 
   const execution = fakeExecutionFactoryV2({ parentArn: undefined });
   await executionsModel.create(execution);
+  const execution2 = fakeExecutionFactoryV2({ parentArn: undefined });
+  await executionsModel.create(execution2);
 
   t.teardown(async () => {
     logSpy.restore();
     await executionsModel.delete({ arn: execution.arn });
+    await executionsModel.delete({ arn: execution2.arn });
   });
 
-  await migrateExecutions(process.env, t.context.knex);
+  await migrateExecutions(
+    process.env,
+    t.context.knex,
+    {
+      loggingInterval: 1,
+      parallelScanLimit: 1,
+      parallelScanSegments: 2,
+    }
+  );
   t.true(logSpy.calledWith('Batch of 1 execution records processed, 1 total'));
+  t.true(logSpy.calledWith('Batch of 1 execution records processed, 2 total'));
```

Comment:
Nit: there probably should be a cleanup of the migrated record here given...... 

Nevermind, it's in the afterEach.  Nice work :100: